src/main/java/bio/terra/cli/service/GoogleGcsBuckets.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cli.service;
+
+import bio.terra.cli.service.utils.CrlUtils;
+import bio.terra.cloudres.google.storage.BlobCow;
+import bio.terra.cloudres.google.storage.BucketCow;
+import bio.terra.cloudres.google.storage.StorageCow;
+import com.google.api.gax.paging.Page;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.util.stream.StreamSupport;
+
+public class GoogleGcsBuckets {
+  private final StorageCow storageCow;
+
+  public GoogleGcsBuckets(GoogleCredentials credentials) {
+    this.storageCow = CrlUtils.createStorageCow(credentials);
+  }
+
+  /**
+   * Get an exact (slow) count of the objects in a bucket.
+   *
+   * @param bucketName - name of the bucket
+   * @return count
+   */
+  public long getBucketObjectCount(String bucketName) {
+    BucketCow bucketCow = storageCow.get(bucketName);
```

Comment:
For this, I kinda want to say "100+" items or something instead of keeping on paging through. There's a cost to listing the bucket and it could be significant for very very large buckets IIRC.

Reply:
That seems reasonable, I would think that an unexpected "100+ items" should send someone off to the console or `gsutil` to find out more.

Reply:
Agreed there's no need to list huge numbers of items. Perhaps you can add a `limit` argument to this method.

Reply:
good idea